docker-compose.dev.yml

```diff
@@ -1,6 +1,6 @@
 services:
   devdb:
-    image: postgres:15
+    image: docker.io/postgres:15
```

Comment:
Is this a different Image or the same? If it's different then what is the difference?

Reply:
It's a more systematic name of the same image.